docs/assets/C#NamingGuidelinesTable.md

```diff
@@ -0,0 +1,15 @@
+# C# Naming Conventions - Quick Reference Table
+
+| Kind                                 | Naming Convention |  Example                                                           |
+| ------------------------------------ | ----------------- | ------------------------------------------------------------------ |
+| Classes                              | PascalCase        | `class Car{}`                                                      |
+| Types and Namespaces                 | PascalCase        | `namespace VehicleManufacturer{}`                                  |
+| Parameters                           | camelCase         | `public Car(int odometerMileage, string manufacturer)`             |
+| Methods                              | PascalCase        | `public void StartEngine()`                                        |
+| Properties                           | PascalCase        | `public double FuelLevel { get; set; }`                            |
+| Local Variables                      | camelCase         | `public int yearManufactured;`                                     |
+| Local Functions                      | PascalCase        | `private string CalculateMilesUntilEmpty(double fuelLevel)`        |
+| Fields                               | _PascalCase       | `public string _Day;`                                              |
+| Enum Members                         | PascalCase        | `enum Status { Operational, Broken, InShop }`                      |
```

Comment:
nit: Enums should also have a default 0th member. So if we are going to list a few examples we should follow that guideline as well. Perhaps something like:
```suggestion
| Enum Members                         | PascalCase        | `enum Status { Unknown, Operational, Broken, InShop }`                      |
```

Reply:
I like it